src/modules/drawing/useKmlDataManagement.composable.js

```diff
@@ -18,19 +18,33 @@ export default function useSaveKmlOnChange(drawingLayerDirectReference) {
     const drawingLayer = inject('drawingLayer', drawingLayerDirectReference)
 
     const store = useStore()
+    const online = computed(() => store.state.drawing.drawingOverlay.online)
     const projection = computed(() => store.state.position.projection)
     const activeKmlLayer = computed(() => store.getters.activeKmlLayer)
     const availableIconSets = computed(() => store.state.drawing.iconSets)
+    const temporaryKmlId = computed(() => store.state.drawing.drawingOverlay.kmlId)
```

Comment:
dito

Reply:
I should have read my code after the refactoring of the showDrawingOverlay from ui store into drawing store 🙈.
I think I got it right know